stellargraph/connector/neo4j/graph.py

```diff
@@ -58,25 +66,94 @@ def nodes(self):
         result = self.graph_db.run(node_ids_query)
         return [row["node_id"] for row in result.data()]
 
-    def node_features(self, node_ids):
+    def cache_nodes(self, dtype="float32"):
```

Comment:
Hmm I kind of wanted to avoid going beyond the scope of "make the Neo4j GraphSAGE related classes use Neo4jStellarGraph" as much as possible in this PR. Yeah I agree a smarter caching approach would be nice, but I created this function so that it lets us approximately maintain existing behaviour of having node features in memory. Thoughts?

Reply:
Sounds reasonable, with a name change 👍